crm_mon xml outputs cluster timing settings in consistent unit.

epenchev opened the pull request:
Adding two additional options (stonith-timeout and priority-fencing-delay) to the crm_mon xml output.
Outputting cluster timing settings in a consistent unit (milliseconds), so they can easily get parsed from monitoring tools not having access to the cluster configuration.

kgaillot commented:
> Can one of the admins verify this patch?

add to whitelist

kgaillot commented:
Thanks, this is a good idea.

I recently merged a 2.12 schema update, so this should use schema 2.13.

Since we are normalizing the values to ms, I'd rather output it like stonith-timeout-ms="60000" so the schema can restrict it to integers and users can parse it more easily.

This raises the more general question of which options should be listed, in which formats. text/console output is easily cluttered so it makes sense not to add any new ones there. HTML is questionable -- it would be nice if we could make it a collapsible dropdown (which would require mucking with the default CSS). I'm fine with adding just some options, but we should consider what criteria we should use to decide which options get listed.

epenchev commented:
Thanks for the review.

I've changed the output to correspond with the proposed format, stonith-timeout-ms="60000" riority-fencing-delay-ms="0".
Schema is changed to restrict the values to integers and updated to version 2.13.

kgaillot commented:
Thanks!
